src/components/LabelMapper/LabelMapper.tsx

```diff
@@ -0,0 +1,30 @@
+import { Label as LabelPF, LabelProps as LabelPropsPF } from '@patternfly/react-core';
+
+import { EmptyStateSymbol } from 'components/EmptyStateSymbol/EmptyStateSymbol';
+
+import { uiLogger } from 'services/uiLogger';
+
+interface ILabelMapperItem {
+  text: string;
+  color: LabelPropsPF['color'];
+}
+
+export type ILabelMapper<Type> = {
+  [key in Type as string | number | symbol]: ILabelMapperItem;
```

Comment:
`as string | number | symbol` are all parts required?

Reply:
This was just all possible types of keys. Removed `symbol` as may not use in our cases.

Reply:
Then why `number` is used?

Reply:
if we do not have any enum types that take `number` as the type, then we can remove it.

Reply:
Do we have any enum types with `number`?

Reply:
I can not find any in `/pnc-api-types-ts/` for now, so I just removed it in last commit.

Reply:
done